target/linux/ath79/dts/qca9563_comfast_cf-e375ac.dts

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+	model = "COMFAST CF-E375AC";
+	compatible = "comfast,cf-e375ac", "qca,qca9563";
+
+	aliases {
+		led-boot = &led_lan;
+		led-failsafe = &led_lan;
+		led-running = &led_lan;
+		led-upgrade = &led_lan;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_lan: led_lan {
+			label = "cf-e375ac:green:lan";
+			gpios = <&gpio 17 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_wlan5g {
+			label = "cf-e375ac:red:wlan5g";
+			gpios = <&gpio 15 GPIO_ACTIVE_HIGH>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		led_wlan2g {
+			label = "cf-e375ac:blue:wlan2g";
+			gpios = <&gpio 16 GPIO_ACTIVE_HIGH>;
+			linux,default-trigger = "phy1tpt";
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+				linux,code = <KEY_RESTART>;
+				gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+				debounce-interval = <60>;
+			};
+		};
+
+	watchdog {
+		compatible = "linux,wdt-gpio";
+
+		gpios = <&gpio 6 GPIO_ACTIVE_LOW>;
+		hw_algo = "toggle";
+		hw_margin_ms = <1200>;
+		always-running;
+	};
+};
+
+&uart {
+	status = "okay";
+};
+
+&gpio {
+	status = "okay";
+};
+
+&usb0 {
+	status = "okay";
+};
+
+&pcie {
+	status = "okay";
+};
+
+&spi {
+	status = "okay";
+
+	num-cs = <1>;
+
+	flash@0 {
+		compatible = "winbond,w25q128","jedec,spi-nor";
```

Comment:
Add a space after the comma. Is the first string "winbond,w25q128" necessary?

Reply:
I've recompiled without the first string and it seems to work just fine.
Should I remove it?

Reply:
Yes, please remove it. Auto-detection is preferred.